registry/extension/oci/discover.go

```diff
@@ -29,7 +30,7 @@ func (eh *extensionHandler) getExtensions(w http.ResponseWriter, r *http.Request
 
 	// remove the oci extension so it's not returned by discover
 	for i, e := range enumeratedExtensions {
-		if e.Name == namespaceName {
+		if e.Name == fmt.Sprintf("_%s", namespaceName) {
```

Comment:
nit: `fmt.Sprintf("_%s", namespaceName)` is evaluated multiple times in the for loop, which is not efficient.